dockerfile-buildx-to-build.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+
+for i in $(find . -name Dockerfile.buildx -print); do
```

Comment:
You could leave off the -print if you want.

Reply:
I prefer to be explicit.